WordPress/WordPressTest/Services/PostServiceWPComTests.swift

```diff
@@ -370,10 +289,6 @@ private class PostServiceRESTMock: PostServiceRemoteREST {
     private(set) var invocationsCountOfAutoSave = 0
     private(set) var invocationsCountOfUpdate = 0
 
-    override func getPostWithID(_ postID: NSNumber!, success: ((RemotePost?) -> Void)!, failure: ((Error?) -> Void)!) {
```

Comment:
My options were to either try to update the tests that will be removed soon to use the new APIs or to remove them, and I chose to remove them now rather waste time updating them.

It's easier to mock on the HTTP level, like the new tests do, and the HTTP-based mocks cover more area.